src/Opdex.Platform.WebApi/Controllers/VaultGovernancesController.cs

```diff
@@ -37,6 +37,22 @@ public VaultGovernancesController(IApplicationContext context, IMapper mapper, I
         _mediator = mediator ?? throw new ArgumentNullException(nameof(mediator));
     }
 
+    /// <summary>Get Vaults</summary>
+    /// <remarks>Retrieves vaults.</remarks>
+    /// <param name="filters">Filter parameters.</param>
+    /// <param name="cancellationToken">Cancellation token.</param>
+    /// <returns>Vault paging results.</returns>
+    /// <response code="200">Vault results returned.</response>
+    /// <response code="400">The request is not valid.</response>
+    [HttpGet]
+    [ProducesResponseType(typeof(VaultGovernancesResponseModel), StatusCodes.Status200OK)]
+    [ProducesResponseType(typeof(ValidationProblemDetails), StatusCodes.Status400BadRequest)]
```

Comment:
[Authorize] is on this entire controller so we may want to add the 401 response types